hometask_day_2/task5.py

```diff
@@ -0,0 +1,20 @@
+my_string = "hello 1 one two three 15 world"
+
+def find_three_words_in_a_row(string):
+    str_to_search = string.split()
+    counter = 0
+    for word in str_to_search:
+
+        if counter == 3:
+            return "There are 3 words in a row"
+
+        try:
+            num = int(word)
+            counter = 0
+        except ValueError:
+            counter += 1
+            continue
```

Comment:
I believe "continue" here is unnecessary. Just forgot to remove it after editing the code.

Reply:
I agree

Reply:
Also, if you find the mistake in your code, just fix it locally the `commit` and `push` again, github will automatically update it in PR